.terraform-version

```diff
@@ -1 +1 @@
-0.11.13
+0.12.19
```

Comment:
0.12.20 is out now. 🙈

Reply:
don't care, wasn't when i started this 😄